src/interface/settings_game.py

```diff
@@ -0,0 +1,146 @@
+        self.main.title("Settings Sudoku Game")
+    
+        #definying the UI option to shown up and edit the values of Output file
+        Label(self.main, text = "Output File Config:").place(x = 15, y= 10)
+        Label(self.main, text = "Path: ").place(x = 15, y = 40)
+        
+        # fill values in the output entry
+        self.fill_values_in_the_output_file_entry()
+        
+        # for future suported types
+        #self.Listofsupportedfilestypes()
+
+        #create a listbox to list all  to list the available algorithms 
+        Label(self.main, text = "Solve Algorithm:").place(x = 15, y = 90)
+        Label(self.main, text = "Algorithm: ").place(x = 15, y = 115)
+        self.show_list_algorithms()
+
+        #Creaate a combo box with levels values
+        Label(self.main, text = " Complexity :").place(x = 15, y= 200)
+        Label(self.main, text = " Level :").place(x = 15, y= 220)
+        
+        #create and fill the combobox
+        self.show_level_combobox()
+
+        Label(self.main, text = "TopLimit: ").place(x = 15, y = 250)
+        Label(self.main, text = "BottonLimit: ").place(x = 15, y = 280)
+        
+        
+        #buttons
+        save_button = Button(self.main, text = " Save ", width = 10, heigh = 2)
+        save_button.place(x = 100, y = 330)
+
+        close_button = Button(self.main, text = " Close ", width = 10, heigh = 2, command = self.quit)
+        close_button.place(x = 200, y = 330)
+
+        self.main.mainloop()
+
+    def fill_values_in_the_output_file_entry(self):
+        """
+        shows the value in output file box.
+        getvaluesoutputfile -- the string value get from "readconfigfile" class to get the Output file settings saved in config file.
+
+        """
+        get_values_output_file = ReadConfigFile(self.file_config).get_output_file()
+        current_path = StringVar()
+        
+        current_path.set(get_values_output_file[0] + '\\' + get_values_output_file[1] + '.' + get_values_output_file[2])
+        
+
+        outputf1 = Entry(self.main, textvariable = current_path, width = 30).place(x = 100, y = 40)
+        
+
+    def list_of_supported_files_types(self):
+        """
+        function will populate all files formats to be supported in the Game
+        """
+
+        list_types = ["txt", "csv", "xml"]  
+        list_types = ttk.Combobox (self.main, state = 'readonly', width = 5)
+        list_types['values'] = (list_types)
+        list_types.current(0)
+        list_types.place(x = 270, y = 65)
+
+
+    def show_list_algorithms(self):
+        """
+        calls getlistofalgorithms function from "readconfigfile" class to get the list of availables algorithms saved in config file.
+        and this info will be populated in a listbox in the UI
+
+        """
+        
+        list_of_algorithms = ReadConfigFile(self.file_config).get_list_of_algorithms()      
+        
+        lst_algorithms = Listbox(self.main, heigh = 4)
```

Comment:
lst_algorithms?? what is the difference with list_of_algorithms?? it wouldn't be better to have listbox_list_algorithms?